### PR TITLE
Add asset tab

### PR DESCRIPTION
**Pull request by sergiojiu747**

### CODE REVIEW

**Comment by sergiojiu747** [comment]
tokentab

File: config/nginx.default.conf [hunk 1]
```diff
@@ -24,7 +24,7 @@ server {
 
     location /api {
         rewrite ^/api/(.*) /$1 break;
-        proxy_pass http://wallet.v.systems:9922;
+        proxy_pass http://wallettestnet.v.systems:9922;
```
Comment (SheldonYS) [inline config/nginx.default.conf]: This change should not be committed.

File: src/constants.js [hunk 2]
```diff
@@ -11,7 +11,7 @@ var __assign = (this && this.__assign) || Object.assign || function(t) {
 };
 export const WALLET_VERSION = '0.1.4'
 export const INITIAL_SESSION_TIMEOUT = 5
-export const NETWORK_BYTE = 'M'.charCodeAt(0);
+export const NETWORK_BYTE = 'T'.charCodeAt(0);
```
Comment (SheldonYS) [inline src/constants.js]: All changes of this file should not be committed.

File: src/constants.js [hunk 3]
```diff
@@ -39,7 +39,7 @@ export const DEFAULT_BASIC_CONFIG = {
     timeDiff: 0
 };
 
-export const NODE_IP = '/api'
+export const NODE_IP = 'http://test.v.systems:9922'
```
Comment (SheldonYS) [inline src/constants.js]: This change should not be committed.

File: package-lock.json [hunk 4]
```diff
@@ -6776,14 +6776,6 @@
...
             "dev": true
         },
-        "json-bignumber": {
-            "version": "1.0.1",
-            "resolved": "https://registry.npmjs.org/json-bignumber/-/json-bignumber-1.0.1.tgz",
-            "integrity": "sha512-3ocoLAUJNsPRmgVJmUGkN6TP5VLxO48mgIaz2xAOS9RppP1S3YSJE0jnpW+uk63CONqJaq0dKlS72wEhmG3XdA==",
-            "requires": {
-                "bignumber.js": "7.2.1"
-            }
-        },
```
Comment (SheldonYS) [inline package-lock.json]: add lib "json-bignumber" back.

File: src/components/home/modals/Receive.vue [hunk 5]
```diff
@@ -131,16 +131,6 @@ export default {
...
-        },
-        copyAddr() {
-            this.$refs.addrToCopy.select()
-            window.document.execCommand('copy')
-            this.$root.$emit('bv::show::popover', 'btn-cpy')
-            this.isCpyDisable = true
-            setTimeout(() => {
-                this.$root.$emit('bv::hide::popover', 'btn-cpy')
-                this.isCpyDisable = false
-            }, 400)
```
Comment (SheldonYS) [inline src/components/home/modals/Receive.vue]: This part of code cannot be removed. Or we'll miss to do copy action after click copy button.

File: src/components/home/elements/TokenRecord.vue [hunk 6]
```diff
@@ -0,0 +1,264 @@
...
+    computed: {
+        txAddressShow() {
+            if (this.txAddress) {
+                const addrChars = this.txAddress.split('')
+                addrChars.splice(6, 23, '******')
+                return addrChars.join('')
+            }
+        },
+        txTitle() {
+            return this.address
```
Comment (SheldonYS) [inline src/components/home/elements/TokenRecord.vue]: This should be tokenID or tokenTicker. It should be load from local storage. If not implemented, just fill with a hard code string first.

File: src/components/home/elements/TokenRecord.vue [hunk 7]
```diff
@@ -0,0 +1,264 @@
...
+      <b-col class="record-detail"
+             cols="auto">
+        <b-row>
+          <b-col class="title">{{ txTitle }}</b-col>
+        </b-row>
+      </b-col>
+      <b-col class="record-blank"></b-col>
+      <b-col cols="auto">
+        <div>
+          <span>{{ formatter(txAmount) }} VSYS</span>
```
Comment (SheldonYS) [inline src/components/home/elements/TokenRecord.vue]: There is no unit symbol for token. Just remove "VSYS".

File: src/components/home/elements/LeaseRecords.vue [hunk 8]
```diff
@@ -37,8 +36,7 @@
       </b-dropdown>
     </div>
     <div class="inherit-height">
-      <div class="scroll"
-           :style="{height: myHeight}">
```
Comment (SheldonYS) [inline src/components/home/elements/LeaseRecords.vue]: Why need remove this code?

File: src/components/home/elements/TokenRecord.vue [hunk 9]
```diff
@@ -117,12 +117,12 @@ export default {
...
-        txTitle() {
-            return this.address
-        },
         txAmount() {
             return 100
-        }
+        },
+        tokenId() {
+            return this.address
+        },
```
Comment (SheldonYS) [inline src/components/home/elements/TokenRecord.vue]: Remove comma here.

**Pull request merged**